src/+energy/calculateEnergyParameters.m

```diff
@@ -30,6 +36,12 @@
     Vaa = AirVariabes.Vaa;
     QL = AirVariabes.QL;
 
+    if ModelSettings.Soilairefc % added by Mostafa
+        QL_h = AirVariabes.QL_h;
+        QL_T = AirVariabes.QL_T;
+        QL_a = AirVariabes.QL_a;
```

Comment:
Please add the names of the variables in front of each variable as inline comments.

Reply:
> Please add the names of the variables in front of each variable as inline comments.

Done